### PR TITLE
[docs] `adapter-cloudflare-workers`: describe how to skip webpack bundling

### LINKED ISSUE #2679: ES target ignored in certain cases [cloudflare-workers]
### Describe the bug

When updating recently to a newer version of SvelteKit and Cloudflare workers adapter I started getting an error when trying to deploy my code with `wrangler publish`
```
👀  ./workers-site/index.js 7390:28
Module parse failed: Unexpected token (7390:28)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|   
|     
>     console.log(request ?? "foobar");
|     
|  
Error: webpack returned an error. You may be able to resolve this issue by running npm install.
```

I use typescript, and I have `target: es2019` specified in my `tsconfig.json`, so it should be compiling down the `??`.
The interesting thing is this only happens under certain conditions. When I use the `??` operator in a `.svelte` file, it gets compiled to es2019 without any issues. But if the code is in `hooks.ts`, it doesn't get compiled down. This used to work just fine, so the issue must be in something changed in the last 2-3 weeks at most.

My current work-around to fix the issue, is to add to `svelte.config.js`: `kit.vite.build.target = 'es2019'`. After I add this, everything builds just fine.

### Reproduction

- Initialise a new SvelteKit demo app with typescript.
- Add some code using an esnext feature like `??` to `hooks.ts`
- Configure cloudflare workers adapter (This might not even be related, but its what I'm using atm)
- Run `npm run build`
- Look inside the `workers-site/index.js` output file, you can see the `??` being left in the code.

### Logs

_No response_

### System Info

```shell
System:
    OS: macOS 11.6
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 1.36 GB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 17.0.1 - /usr/local/bin/node
    npm: 8.1.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 94.0.4606.81
    Safari: 15.0
  npmPackages:
    @sveltejs/adapter-cloudflare-workers: ^1.0.0-next.20 => 1.0.0-next.25 
    @sveltejs/kit: next => 1.0.0-next.189 
    svelte: ^3.38.3 => 3.44.0
```

### Severity

serious, but I can work around it

### Additional Information

I'm unsure if this is a bug in Svelte, SvelteKit, Cloudflare worker adapter, or something completely different. All I know is I never had this issue until just recently, and my code has been using esnext features all along, and I only just started having issues after my most recent upgrade.

### PR DESCRIPTION
**Pull request by half2me**
Updated information on `wrangler.toml` configuration for the cloudflare workers adapter. fixes #2679

### CODE REVIEW

File: packages/adapter-cloudflare-workers/README.md [hunk 1]
```diff
@@ -38,12 +38,21 @@ Now you should get some details from Cloudflare. You should get your:
...
 ```toml
 account_id = 'YOUR ACCOUNT_ID'
 zone_id    = 'YOUR ZONE_ID' # optional, if you don't specify this a workers.dev subdomain will be used.
 site = {bucket = "./build", entry-point = "./workers-site"}
+
+type = "javascript"
+compatibility_date = "2021-10-22"
+
+[build]
+command = ""
```
[the remark this answers is not in the document]
Comment (half2me) [reply packages/adapter-cloudflare-workers/README.md:52]: As long as the docs are clear about the difference between building and publishing. At the end of the day, users are free to put whatever command they wish in the toml file, as long as they understand the consequences.

File: packages/adapter-cloudflare-workers/README.md [hunk 2]
```diff
@@ -38,12 +38,21 @@ Now you should get some details from Cloudflare. You should get your:
...
-Then configure your sites build directory and your account-details in the config file:
+Then configure your sites build directory, your account-details and some other options in the config file. You may optionally specify a build command which will run before publishing (For example: `npm run build`) or just leave it empty. For all the available options, take a look at the [wrangler.toml](https://developers.cloudflare.com/workers/platform/sites/configuration) documentation on Cloudflare.
 
 ```toml
 account_id = 'YOUR ACCOUNT_ID'
 zone_id    = 'YOUR ZONE_ID' # optional, if you don't specify this a workers.dev subdomain will be used.
 site = {bucket = "./build", entry-point = "./workers-site"}
+
+type = "javascript"
+compatibility_date = "2021-10-22"
```
Comment (benmccann) [inline packages/adapter-cloudflare-workers/README.md:49]:
```suggestion
```

File: packages/adapter-cloudflare-workers/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (benmccann) [reply packages/adapter-cloudflare-workers/README.md:51]:
```suggestion
# Assume it's already been built. You can make this "npm run build" to ensure a build before publishing
command = ""
```

File: packages/adapter-cloudflare-workers/README.md [hunk 3]
```diff
@@ -38,12 +38,21 @@ Now you should get some details from Cloudflare. You should get your:
 
 Get them by visiting your Cloudflare-Dashboard and click on any domain. There, you can scroll down and on the left, you can see your details under **API**.
 
-Then configure your sites build directory and your account-details in the config file:
+Then configure your sites build directory, your account-details and some other options in the config file. You may optionally specify a build command which will run before publishing (For example: `npm run build`) or just leave it empty. For all the available options, take a look at the [wrangler.toml](https://developers.cloudflare.com/workers/platform/sites/configuration) documentation on Cloudflare.
```
[the remark this answers is not in the document]
Comment (benmccann) [reply packages/adapter-cloudflare-workers/README.md:41]:
```suggestion
Then configure your sites build directory and your account-details in the config file:
```

**Pull request merged**